exercises/robot-name/RobotNameTest.cs

```diff
@@ -30,4 +31,20 @@ public void Can_reset_the_name()
         robot.Reset();
         Assert.NotEqual(originalName, robot.Name);
     }
+
+    [Fact(Skip = "Remove to run test")]
+    public void After_reset_the_name_is_valid() {
+        robot.Reset();
+        Assert.Matches(@"^[A-Z]{2}\d{3}$", robot.Name);
+    }
+
+    [Fact(Skip = "Remove to run test")]
+    public void Robot_names_are_unique() {
+        var names = new HashSet<string>();
+        for (int i = 0; i < 10_000; i++) {
+            Assert.False(names.Contains(robot.Name), $"Name {robot.Name} was already used ({i})");
```

Comment:
I think there is an `Assert.DoesNotContain` method that you could use here. Note that we don't use an explicit assert message anywhere else in the test suites, so I wouldn't mind removing it here (especially as `Assert.DoesNotContain` will be quite descriptive).